w3af/plugins/audit/global_redirect.py

```diff
@@ -112,8 +124,7 @@ def _domains_are_equal(self, redir_url, test_url):
         """
         try:
             redir_domain = urlparse.urlparse(redir_url).netloc
-            test_domain = urlparse.urlparse(test_url).netloc
-            return redir_domain == test_domain
+            return redir_domain.endswith(self.TEST_DOMAIN)
```

Comment:
I don't like this, what if the site redirects to `http://target.com.w3af.org/`? Note the trailing slash

Reply:
`redir_domain` is processed by `urlparse.urlparse()`, so there should not a trailing slash

Reply:
Ahh you are right, sorry about that.